### DISCUSSION TITLE
Mirth Process user account

### DISCUSSION THREAD

**Discussion by hubba002** [nextgenhealthcare/connect#6325] [Q&A] [answered]
Is there a way to configure the Mirth process (daemon)  to run under a specific user account on a Windows Server?  I need to do this so the account it is running using an Active Directory account to the Mirth account can access our NAS file shares.

**Comment by pacmano1** [comment, marked as answer]
Yes, just set the user account via Service Manager.  Do make sure the directory where mirth is installed has the correct permissions for this user.  Fast way is just "full access" on the directory, subdirectories and files.
